### DISCUSSION TITLE
GPS, self-test, every Take-off at the end of the runway

### DISCUSSION THREAD

**Discussion by Riggie** [flybywiresim/aircraft#4869] [Help]
Regarding A320NX Dev

Hi Everyone,
When I launch the flight to start at the end of the runway, I get all the screens going through a self-test,  then the GPS screen comes up with ADIRS failure/ then self-test will take 7mins, basically,  it is as if the Aircraft is starting from Cold & Dark but at the runway ready for take-off?
This can occur about 50% of the time, I now find it quicker to restart the flight and it sometimes fixes the problem.
Also, I am asked to enter the Squawk about 50% of the time?

Is there any way to remove FBW cleanly and start again from fresh or, is just removing it from the Community folder the only option?

**Comment by ExampleWasTaken** [comment]
This is a bug in the simulator and nothing we can fix unfortunately. 

As for your second question, to completely fresh-install the A32NX, you have to delete the `flybywire-aircraft-a320-neo` folder from your community folder and install it again through the [official installer](https://api.flybywiresim.com/installer).

**Reply by Riggie** [reply]
Thanks for the reply much appreciated.
